src/PerformanceTests/Common/Scenarios/BaseRunner.v5.cs

```diff
@@ -83,15 +83,8 @@ void OnCriticalError(string errorMessage, Exception exception)
     {
         try
         {
-            try
-            {
-                Log.Fatal("OnCriticalError", exception);
-                Session.CloseWithSuppress().ConfigureAwait(false).GetAwaiter().GetResult();
-            }
-            finally
```

Comment:
Lgo4net also has a shutdown method.

Reply:
why would we need to shutdown the logger here? Isn't that done by the finally block in `Program.cs`?

Reply:
The one in Main isn't called when doing `Environment.FailFast`, see it as a process kill.